### ISSUE TITLE
Missing definition of var?

### ISSUE THREAD

**Issue by brettha** [editorscut/ec010functionalkickstart#20]
Functional Programming Kickstart: Release 0.6
Chapter: 6
Section: 1

Description:
In the subtitle "The world of Arrays"
Hi Daniel, I'm enjoying my Memorial Day reading your book.  Just finished the map States section and looking forward to flatMap.

The book says to write "ints.map(cardInFreshDeck)" but "ints" is not defined.  I see in the FlatMapFinal.playground it is defined as "let ints = [1, -49, 38, 125, 24]".  It would be helpful to add that line of code in your book.

**Comment by editorscut** [comment]
I'm sorry - I hate making mistakes like that because it makes it so hard to follow along. Thank you for sending this in - I've fixed this for today's release.

Best,

Daniel
